rs/registry/canister/tests/recover_subnet.rs

```diff
@@ -670,216 +448,6 @@ fn test_recover_subnet_gets_schnorr_keys_when_needed() {
     test_recover_subnet_gets_chain_keys_when_needed(key_id);
 }
 
-// TODO[NNS1-3022]: Delete this test once `RecoverSubnetPayload.ecdsa_config` is obsolete.
-#[test]
-fn test_recover_subnet_without_ecdsa_key_removes_it_from_signing_list_legacy() {
```

Comment:
same question for the rest of the tests - already a chain key equivalent?

Reply:
Yes, the chain key cousins of this particular test are called 

`test_recover_subnet_without_ecdsa_key_removes_it_from_signing_list` 

and 

`test_recover_subnet_without_schnorr_removes_it_from_signing_list`.